modules/renter/contractor/contracts.go

```diff
@@ -343,20 +341,41 @@ func (c *Contractor) threadedContractMaintenance() {
 	// Renew any contracts that need to be renewed.
 	c.mu.RLock()
 	var renewSet []types.FileContractID
+	refundSet := make(map[types.FileContractID]struct{})
```

Comment:
Can you add a comment here explaining the purpose of `refundSet`? It's not immediately clear.